app/src/lang/translations/mn-MN.yaml

```diff
@@ -21,6 +57,7 @@ create_user: Хэрэглэгч үүсгэх
 create_webhook: Webhook үүсгэх
 invite_users: Хэрэглэгч урих
 email_examples: "admin{'@'}example.com, user{'@'}example.com..."
+url_example: "https://jishee.com"
```

Comment:
just curious, are URLs typically vetted? just a random thought popped up in my mind when I saw this.